docs/content/how-to/aws/troubleshooting.md

```diff
@@ -0,0 +1,111 @@
+
+4. Now we need to delete the OVN pods
+```
+oc --kubeconfig=${HC_KUBECONFIG} delete pod -n openshift-ovn-kubernetes --all
+```
+
+Eventually the HostedCluster will start self healling and the ClusterOperator will come back.
+
+**Cause:** The cause of this issue it's after the HostedCluster Migration the KAS (Kubeapi Server) uses the same DNS name, but it points to different load balancer in AWS platform. Sometimes OVN does not behaves correctly facing this situation.
+
+### The migration get blocked, in ETCD recovery
+
+The context around it's basically "I've edited the HostedCluster adding the `ETCDSnapshotURL` but the modification disappear and does not continue".
+
+The first simpthom it's the status of the Hypershift Operator pod, in this case the pod is usually in `CrashLoopBackOff` status.
+
+To solve this issue we need to:
+
+1. Kill the Hypershift Operator pod
+```
+oc delete pod -n hypershift -lapp=operator
+```
+
+2. Continue editing the pod, in order to add the `ETCDSnapshotURL` to the HostedCluster spec.
+
+3. Now the ETCD pod will raise up using the snapshot from S3 bucket.
+
+**Cause:** This issue happens when the Hypershift operator it's down and the HostedCluster controller cannot handle the modifications in the objects which belongs to him.
+
+### The nodes cannot join the new HostedCluster and keeps in the older one
+
+We have 2 paths to follow and depends if [this code](https://github.com/openshift/hypershift/pull/2265) it's in your Hypershift Operator.
+
+#### The PR is merged and my Hypershift Operator has that code running
+
+If that's the case, you need to make sure your HostedCluster is paused:
+```
+oc get hostedcluster -n <HC Namespace> <HC Name> -ojsonpath={.Spec.pausedUntil}
+```
+
+If this commands does not give you any output, make sure you've followed properly the "[Disaster Recovery](https://hypershift-docs.netlify.app/how-to/aws/disaster-recovery/)" procedure. More concretelly pausing the HostedCluster and NodePool.
+
+Even if it's paused and is still in that situation, please **continue to the next section** because it's highly probable that you don't have the code which manages this situation properly.
+
+#### The PR is not merged or my Hypershift Operator has not that code running
+
+If that's not the case, the only way to solve it is executing the teardown of the old HostedCluster prior the full restoration in the new Management cluster. Make sure you already has all the Manifests and the ETCD backed up.
+
+Once you followed the Teardown procedure of the old HostedCluster, you will see how the migrated HostedCluster begins to self-recover
+
+**Cause:** This issue occurs when the old HostedCluster have a conflict with the AWSPrivateLink object. The old one is still running and the new one cannot handle it because the `hypershift.local` AWS internal DNS entry still points to the old LoadBalancer.
```

Comment:
```suggestion
**Cause:** This issue occurs when the old HostedCluster has a conflict with the AWSPrivateLink object. The old one is still running and the new one cannot handle it because the `hypershift.local` AWS internal DNS entry still points to the old LoadBalancer.
```